submissions/dummy_submission/the-nuclear-options-v1.0.md

```diff
@@ -0,0 +1,174 @@
+
+In terms of where to build the nuclear plant as a whole, it is quite beneficial to build near or on water, because the plant needs tons of it. Being close to uranium is optional because necesary amounts can be easil transported via trains.
+
+When placing down the reactors, we already know about harnessing the neighbor bonus and  calculating it using SRE diagrams. Given that we want to connect as many neighbors as possible, it would make sense to just have a huge spaceless grid of reactors, but the disadvantage of that is there being no space to set up a system for automatically refueling any of the reactors in the middle, unless you [a mod for that](https://mods.factorio.com/mod/logistics-reactor-next). Hence the most popular reactor arrangement is 2xN, where all reactors except for the corner ones have 3 neighbors, hence each has at least 1 open side for inserter access. However, you can get as creative as you want and leave 2xN behind once you have a steady supply of U-235 such that the neighbor bonus is not so critical anymore.
+
+Your other big concern is fluid flow, and the main constraint is the heat flow along heat pipes. The farther away the heat gets from the reactors, the lower the maximum temperature the pipes can have. Since reactors temperatures cannot exceed 1000C, heat pipes beyond a certain distance become too cold to operate heat exchangers. As [the wiki shows](https://wiki.factorio.com/Tutorial:Nuclear_power#Heat_pipes) and [this post effectively visualizes](https://www.reddit.com/r/factorio/comments/ge4y6c/heat_pipe_throughput_and_a_bonus_note_on_parallel/),  a single pipe can heat a row of at most 21 heat exchangers, but you can improve the range here by making the pipe thicker and by adding heat exchangers on both sides of it instead of one side.  Another tricked used by the community here is placing unfueled nuclear reactors to act as 5x5 heat pipes because this allows travelling the same distance with fewer entities and connecting to more heat exchangers from a single entity. In any case, your best bet is to keep your heat pipes short by keeping heat exchangers nearby. 
+
+As for the other fluids, you want to ensure that the water from your offshore pumps is reaching your heat exhangers without significant [throughput loss in pipelines](https://wiki.factorio.com/Fluid_system#Pipelines), hence make sure to have multiple pipelines along the way. While it is ideal to simply have short pipelines, you can also consider using pumps. The critical number is a maximum of 17 pipe segments between every 2 pumps, in order to maintain the 1200 water/second from an offshore pump. Here it is worth noting that underground pipes pairs at any distance count as only 2 pipe segments. Secondly, you want to make sure the steam generated has enough pipe space to flow from exchangers to turbines without backing up, which is not an issue except for enormous nuclear plants and can always be solved again using more pipes or adding pumps. 

```

Comment:
do you have idea if connecting multiple offshore pumps to a unique pipe can increase the distance? I remember myself placing multiple offshore pumps on a single pipe and never had issues with water being able to reach the exchangers with sufficient flowrate

Reply:
I'll need to test this one